src/http/controllers/controller.dependents.ts

```diff
@@ -91,4 +91,70 @@ export module DependentsController {
             );
         });
     }
+
+    /**
+     * ! Create new dependet for a customer
+     * * DanBaDo - 2021-12-29
+     * @param req {Request}
+     * @param res {Response}
+     */
+    export const _newDependentForCustomer = async (req, res)=>{
+        const data: newDependentForCustomerDTO = req.body;
+        const customers = data.customersIds;
+        const dependent = data.dependent;
+        DependentsActions.newDependent(
+            dependent.name,
+            dependent.phone,
+            dependent.direction,
+            (err: string, newDependent: Dependent) => {
+                if (err) {
+                    _handleResponse(
+                        { statusCode: 500, message: 'Server response cannot be processed' },
+                        res
+                    )
+                }
+                customers.forEach(
+                    (customerId) => {
+                        DependentsActions.linkDependentToCustomer(
+                            newDependent.id,
+                            customerId,
+                            (err)=>{
+                                if (err) {
+                                    _handleResponse(
+                                        {
```

Comment:
I would like to avoid nesting ```.linkDependentToCustomer``` in ```.newDependent``` if it's possible.

Reply:
It's for me or for @Alcazar87 ?